terraform/helm/aptos-node/values.yaml

```diff
@@ -162,6 +171,15 @@ fullnode:
     test: {}
     validator_network: {}
     failpoints: {}
+  statefulset:
+    # -- To do: fullnode statefulset related property should be pulled here
```

Comment:
nit: could you please remove the "To do"? I guess you don't mean to leave a todo item here?

Reply:
removed `To do`